workshop2/demo-server/main.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018- The Pixie Authors.
```

Comment:
that does not belong with pixie!
you should give me the credits

Reply:
Copied it from your workshop

Reply:
so please remove from both of them